content/administration/odoo_sh/getting_started/first_module.rst

```diff
@@ -414,21 +421,31 @@ The platform will then create a new build for the branch *feature-1*.
 .. image:: first_module/firstmodule-test-addachange-build.png
   :align: center
 
-Once you tested your changes, you can merge your changes in the production branch, for instance by drag-and-dropping the
-branch on the production branch in the Odoo.sh interface. As you increased the module version in the manifest,
-the platform will update the module automatically and your new field will be directly available.
-Otherwise you can manually update the module within the apps list.
+Once you tested your changes, you can merge your changes in the production branch, for instance by
+drag-and-dropping the branch on the production branch in the Odoo.sh interface. As you increased the
+module version in the manifest, the platform will update the module automatically and your new field
+will be directly available. Otherwise you can manually update the module within the apps list.
 
 Use an external Python library
 ==============================
 
 If you would like to use an external Python library which is not installed by default,
 you can define a *requirements.txt* file listing the external libraries your modules depends on.
 
+.. note::
+   - It is not possible to install or upgrade system packages on an Odoo.sh database (e.g., apt
+     packages). However, under `specific conditions
+     <https://www.odoo.sh/faq#install_dependencies>`_, packages can be considered for installation.
+     This also applies to **Python modules** requiring system packages for their compilation, and
+     **third-party Odoo modules** (`incompatible modules
+     <https://www.odoo.sh/faq#install_dependencies>`_).
+   - **PostgreSQL extensions** are not supported on Odoo.sh.
+   - For more information, consult our `FAQ <https://www.odoo.sh/faq#install_dependencies>`_.
```

Comment:
Is it really useful to reference three times the same link ?
I expected to land on different pages when i was clicking on the three links :thinking: